_posts/2020-04-09-status-update.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "8th April 2020: IAC Team Status Update"
+author: michaeltlombardi
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+  - testing
+  - dsc
+  - powershell
+  - windows
+---
+
+The IAC team has been hard at work over the past week.
+Here's a brief overview of what we've been up to:
+
+## New Module Releases (IAC Team)
+
+Let's kick things off by highlighting a new module release:
+
+- [puppetlabs/acl](https://forge.puppet.com/puppetlabs/acl) (v3.1.1)
+
+## Community Contributions
+
+As always, we work hand-in-hand with the broader Puppet community to get work done.
+This week, we want to take a moment to highlight a few PRs and the people behind them!
+
+First up, a [pull request to the puppetlabs/docker module][community-seriv-pr] by [Sergey Ivanov][community-seriv].
+Sergey's work enabled the module to function transparently on RedHat systems utilizing the [`podman-docker` package!][podman-docker].
+Thank you, Sergey!
+
+We'd like to thank [Atanas Stoyanov][community-amorphina] for [his fix to the Apache module][community-amorphina-pr].
+This PR ensures that the custom ldap options in `/etc/apache2/mods` can actually be applied and loaded, fixing a previously unknown bug!
+Thank you, Atanas!
+
+Finally, big Thanks to [Frank Wall][community-fraenki] for [his PR][community-fraenki-pr] that will fix [MODULES-10023][modules-10023], ensuring that the xtrabackup functionality in [puppetlabs/mysql][puppetlabs/mysql] actually works as intended!
+Thank you, Frank!
+
+## Supported Module Litmus Conversion
+
+On the topic of [Litmus][litmus-gem-home], we're continuing to make progress on the conversion of all supported modules to the framework:
+
+- [puppetlabs/reboot][puppetlabs/reboot] is completed, running against all 33 of the supported platforms!
```

Comment:
```suggestion
- [puppetlabs/reboot][puppetlabs/reboot] is nearing completion, running against all 33 of the supported platforms!
```
😢

Reply:
If we merge slowly enough, it might be true!

Reply:
😆 True that. I just think it would be a bit too tight to state this week, but, I certainly would say 100% guaranteed for next week's blog post (NOW I HAVE THE PRESSURE ON ME!).